### PR TITLE
ICRV: Intro Message Styling & Copy

### PR DESCRIPTION
**Pull request by ahsan-ul-haq**
ECOM-2140

Updated the style of the ICRV intro message. 
screen shot 2015-08-28 at 6 53 33 pm

### CODE REVIEW

**Comment by ahsan-ul-haq** [comment]
@zubair-arbi @awais786 @aamir-khan please review

**Comment by ahsan-ul-haq** [comment]
@srpearce Could please review the strings used.

**Comment by zubair-arbi** [comment]
lgtm :+1:

File: edx_reverification_block/xblock/static/html/reverification.html [hunk 1]
```diff
@@ -73,3 +76,11 @@
     </div>
   </div>
 </div>
+<div class="faq-reverification-block">
+  <h4>
+    {% trans "See Also" %}
+  </h4>
+  <a class="faq-link">
+    {% trans "Frequently asked questions about verification and earning college credit" %}
```
Comment (srpearce) [inline edx_reverification_block/xblock/static/html/reverification.html]: college --> academic

**Comment by srpearce** [comment]
Just one change - otherwise :+1:

**Pull request merged**